db/snapshot/backup_task.cc

```diff
@@ -103,51 +108,139 @@ future<> backup_task_impl::upload_component(sstring name) {
     }
 }
 
+future<> backup_task_impl::list_snapshot_dir() {
+    auto snapshot_dir_lister = directory_lister(_snapshot_dir, lister::dir_entry_types::of<directory_entry_type::regular>());
```

Comment:
process_snapshot_dir?

Reply:
can do

Reply:
done in v3 ([7ab3391](https://github.com/scylladb/scylladb/commit/7ab3391f15e68adfff6187385be438b96bb3d57e))